### PR TITLE
Add event reason for update and delete

### PR DESCRIPTION
**Pull request by saumanbiswas**

### CODE REVIEW

**Comment by TamalSaha** [comment]
When do you use `SuccessfulDelete`?

**Comment by aerokite** [comment]
When deleting monitoring system

File: pkg/eventer/event_recorder.go [hunk 1]
```diff
@@ -8,34 +8,40 @@ import (
...
+	EventReasonInvalidUpdate           string = "InvalidUpdate"
+	EventReasonResuming                string = "Resuming"
+	EventReasonSnapshotFailed          string = "SnapshotFailed"
+	EventReasonStarting                string = "Starting"
+	EventReasonSuccessfulCreate        string = "SuccessfulCreate"
+	EventReasonSuccessfulPause         string = "SuccessfulPause"
+	EventReasonSuccessfulMonitorAdd    string = "SuccessfulMonitorAdd"
+	EventReasonSuccessfulMonitorDelete string = "SuccessfulMonitorDelete"
+	EventReasonSuccessfulMonitorUpdate string = "SuccessfulMonitorUpdate"
+	EventReasonFailedToAddMonitor      string = "MonitorAddFailed"
```
Comment (TamalSaha) [inline pkg/eventer/event_recorder.go]: @aerokite @saumanbiswas , should this be `FailedMonitorAdd` ?

Comment (aerokite) [inline pkg/eventer/event_recorder.go]: I have used only Failed for all failed like kubernetes

Comment (aerokite) [inline pkg/eventer/event_recorder.go]: Also, I suggest to rename "SuccessfulMonitorAdd" to "MonitorAdded". What do you think? @TamalSaha

Comment (TamalSaha) [inline pkg/eventer/event_recorder.go]: I think we should be consistent. So, I think it should be `SuccessfulMonitorAdd`, `FailedMonitorAdd`

**Pull request merged**